source/common/router/router_ratelimit.cc

```diff
@@ -38,10 +38,11 @@ bool RequestHeadersAction::populateDescriptor(const Router::RouteEntry&,
                                               const Http::HeaderMap& headers,
                                               const Network::Address::Instance&) const {
   const Http::HeaderEntry* header_value = headers.get(header_name_);
+
+  // If header is not present and skip_if_absent_ is true, ignore the descriptor
```

Comment:
This comment is still not correct. Returning has no bearing on whether skip_if_absent_ is true?